pallets/identity/src/lib.rs

```diff
@@ -295,6 +296,14 @@ decl_module! {
                 |_| Empty
             );
 
+            StorageIterator::<OldDidRecord<T::AccountId>>::new(b"Identity", b"DidRecords")
+                .drain()
+                .map(|(key, old)|  (key, DidRecord {
+                    primary_key: old.primary_key,
+                    secondary_keys: old.secondary_keys,
+                }))
+                .for_each(|(key, new)| put_storage_value(b"Identity", b"DidRecords", &key, new));
```

Comment:
@CJP10 storage prefix is `identity` not `Identity`. Ideally, it should be `Identity` but we can keep `identity` for now as I don't want to garbage the existing data on the testnet. For mainnet we will change this.

Reply:
@satyamakgec it looks like other upgrades above are using `Identity`. Should those been changed also or is this change not needed?

Reply:
Yeah, we need to change every other place as well. We did fix it for the Alcyone launch but never port it back so it remains left in the develop branch.

Reply:
Okay sounds good

Reply:
@satyamakgec It might make more sense to just upgrade the module name in this PR. Why leave it any longer?

Reply:
I think this would screw up the storage though?

I will add a story to normalise module names before the mainnet, but for Alcyone I think we need to keep with the current names.

Reply:
Yeah, for now, we want to keep the same but for mainnet we will revisit and make sure that we will have consistency in the storage prefix.